File: javascript/paybill.js
var siteloc = "http://localhost/Paybills";
var scriptloc = "/scripts/";
function paybill(accountno,pldtacct,amount)
{
  $.ajax({
      url: siteloc + scriptloc + "paybill.py",
      data: {accountno:accountno,
      	pldtacct:pldtacct,
      	amount:amount
             },
      dataType: 'json',
      success: function (res) {
        console.log(res);
      		if (res["resp"] == "OK")
      		{
      			printreceipt(res["receiptNo"]);
      		}
          else if (res["resp"] == "Insufficient Funds")
            alert("Insufficient Funds!!");
          else
            alert("Something went horribly wrong. Try again later.");
      }
    });
}
function fetchaccountinfo(accountno)
{

	$.ajax({
      url: siteloc + scriptloc + "getaccountinfo.py",
      data: {accountno:accountno
             },
      async:false,
      dataType: 'json',
      success: function (res) {
      		console.log(res);
          $("#emailconfirmation").append("<i>"+res[0][2]+"</i>");
      }
    });

}
 

File: scripts/paybill.py
from dosql import *
import cgi
try:
    import json
except ImportError:
    import simplejson as json

def index(req,accountno,pldtacct,amount):
    accountno = cgi.escape(accountno)
    pldtacct = cgi.escape(pldtacct)
    amount = cgi.escape(amount)
    x = doSql()

    balance = x.execqry("select * from accountinfo("+accountno+");",False)
    balance = balance[0][1]

    if int(balance) < int(amount):
        rets = {'resp':'Insufficient Funds','receiptNo':'NONE'}
    else:
        rets = x.execqry("select * from paybill("+accountno+","+pldtacct+","+amount+");",True)
        rets = {'resp':'OK','receiptNo':'1234'}
    return json.dumps(rets)























